client/components/LearnerViewDirective.js

```diff
@@ -249,10 +249,20 @@ tie.directive('learnerView', [function() {
         $scope.submitCode = function(code) {
           SolutionHandlerService
             .processSolutionAsync(prompts[currentPromptIndex], code, language)
-            .then(appendFeedback);
+            .then(setFeedback);
         };
 
-        clearFeedback();
+        $scope.feedbackParagraphs = [
+          'Greetings!',
+          'This set of exercises focuses on string manipulation.',
```

Comment:
My preference would be to not introduce temporary data here, if at all possible, but if it is not possible to move this line into the question/exercise/content file or would require too much work for the Alpha version, then perhaps a TODO comment to remind us to address this later.